### DISCUSSION TITLE
Improve contrast/visibility of focal point indicator

### DISCUSSION THREAD

**Discussion by rungta** [craftcms/cms#8696] [Ideas]
### Description

The focal point indicator that is overlaid on the image and can be dragged around can sometimes be hard to spot depending on the visual contents of the image being edited, and the size of the browser window. E.g.:

![focal point indicator](https://user-images.githubusercontent.com/663677/58859411-5984cc80-86c7-11e9-97dd-8df5d090ab32.png)

![focal point changed](https://user-images.githubusercontent.com/663677/58859410-5984cc80-86c7-11e9-88f9-0630aba2b12b.png)

Increased contrast for this bit of UI would help in quickly spotting:
a) The fact that the focal point indicator is enabled and somewhere on the image, and
b) The current focal point.

### Additional info

- Craft version: 3.1.28

**Comment by jamiewade** [comment]
I have used a technique similar to this and has worked well in the past: http://www.kennethcachia.com/background-check/

**Comment by brandonkelly** [comment]
Ooh that’s pretty cool @jamiewade!
